bin/aurelia-cli.js

```diff
@@ -2,6 +2,15 @@
 'use strict';
 const resolve = require('../lib/resolve');
 
+const nodeVersion = process.versions.node;
+const [major, minor] = nodeVersion.split('.');
+if (major < 8 || (major === '8' & minor < 9)) {
```

Comment:
Doesn't look like major and minor can ever be number, no?

Reply:
As long as nodejs follows semver, it is safe.

Reply:
sh*t, typo '&'

Reply:
surprised the typo didn't fail my tests on various version.

Reply:
```js
[major, minor] = '8.9'.split('.')
(2) ["8", "9"]
major < 8
false
```

Reply:
Ok, I see. I forgot it's valid JS coercion 👍

Reply:
`"8" < 8` is false, what's not right?